impeller/renderer/backend/vulkan/command_encoder_vk.cc

```diff
@@ -178,18 +183,23 @@ bool CommandEncoderVK::Submit(SubmitCallback callback) {
 
   auto command_buffer = GetCommandBuffer();
 
+  size_t end_frame = gpu_tracer_->RecordCmdBufferEnd(command_buffer);
+
   auto status = command_buffer.end();
   if (status != vk::Result::eSuccess) {
+    gpu_tracer_->OnFenceComplete(end_frame);
```

Comment:
Here and below, should we somehow signal to the tracer that it's ending really fast because of a failure?

I'm assuming here we'd get a zero duration, do we want something like a -1 duration?

Reply:
Done. If we encounter an error we should not report for the frame since its probably an all bets are off situation.